ddp-workspace/projects/ddp-testboston/src/style/session-expired.scss

```diff
@@ -1,5 +1,5 @@
 .session-content-section {
-    margin: 5rem 0 0 0;
+    margin: 5rem 0 15rem 0;
```

Comment:
I bit updated the margin for the session expired page. I noticed, that content is too compressed